rocketpool-cli/commands/wallet/rebuild.go

```diff
@@ -54,41 +54,52 @@ func rebuildWallet(c *cli.Context) error {
 		}(customKeyPasswordFile)
 	}
 
+	var enablePartialRebuildValue = false
+	if enablePartialRebuild.Name != "" {
+		enablePartialRebuildValue = c.Bool(enablePartialRebuild.Name)
+	}
+
 	// Log
 	fmt.Println("Rebuilding node validator keystores...")
+	fmt.Printf("Partial rebuild enabled: %s.\n", enablePartialRebuild.Name)
```

Comment:
`enablePartialRebuild.Name` here is equal to `enable-partial-rebuild` per https://github.com/rocket-pool/smartnode/blob/9bd551093d9de7eddb5cc7f985989bdf18c8b585/rocketpool-cli/commands/wallet/utils.go#L61

Did you mean to printf the value of `enablePartialRebuildValue` instead?

Reply:
Thanks for pointing this out. Frankly I found the use of these oddly confusing. For example, recover.go, [it checks if the name is non-empty before using it](https://github.com/rocket-pool/smartnode/blob/v2/rocketpool-cli/commands/wallet/recover.go#L59) but given the definition of the flags, I don't see how it could be empty.

Regardless, I'll definitely update this log.

Reply:
Well that looks like a bug :(

Reply:
Gotcha - once you left that comment I figured that may be the case. I'll take a note.